### PR TITLE
Add OpenAPI specification for import and export saved object api

### PR DESCRIPTION
**Pull request by zhyuanqi**
### Description

Add specification for import and export saved object api

### Issues Resolved

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6719

## Screenshot

Screenshot 2024-05-30 at 3 20 13 PM

Screenshot 2024-05-30 at 6 03 51 PM

## Changelog
- doc: Add specification for import saved object api

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: docs/openapi/saved_objects/saved_objects.yml [hunk 1]
```diff
@@ -520,6 +520,96 @@ paths:
                 - id: 67a9021c-c97e-4499-8150-9722ab44edd4
                   type: visualization
                   fields: ['title', 'fieldFormatMap']
+      responses:
```
Comment (zhyuanqi) [inline docs/openapi/saved_objects/saved_objects.yml:523]: Forgot to add response for bulk_get earlier. Add it now

File: docs/openapi/saved_objects/examples/export_objects_response.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...ipted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"count\":0,\"name\":\"type\",\"type\":\"string\",\"esTypes\":[\"keyword\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"count\":0,\"name\":\"url\",\"type\":\"string\",\"esTypes\":[\"text\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":false,\"readFromDocValues\":false},{\"count\":0,\"name\":\"url.keyword\",\"type\":\"string\",\"esTypes\":[\"keyword\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true,\"subType\":{\"multi\":{\"parent\":\"url\"}}},{\"count\":0,\"name\":\"user\",\"type\":\"string\",\"esTypes\":[\"keyword\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"count\":0,\"name\":\"utc_time\",\"type\":\"date\",\"esTypes\":[\"date\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true}]","timeFieldName":"@timestamp","title":"open*"},"id":"4b3a6220-1e02-11ef-891d-992edd5ce25b","migrationVersion":{"index-pattern":"7.6.0"},"references":[],"type":"index-pattern","updated_at":"2024-05-29T21:27:47.266Z","version":"Wzc0LDFd"}
```
Comment (BionIT) [inline docs/openapi/saved_objects/examples/export_objects_response.yaml:3]: This is a massive example for a response, can we make it smaller?

**Review by zhongnansu — DISMISSED** [review]

File: docs/openapi/saved_objects/saved_objects.yml [hunk 3]
```diff
@@ -520,6 +520,96 @@ paths:
...
+        '200':
+          description: The bulk get request is successful.
+          content:
+            application/json:
+              schema:
+                type: object  
+  /api/saved_objects/_export:
```
Comment (BionIT) [inline docs/openapi/saved_objects/saved_objects.yml:530]: can we change the PR title to match the API we are documenting?

File: docs/openapi/saved_objects/examples/export_objects_response.yaml (hunk 2 above)
Comment (zhyuanqi) [reply docs/openapi/saved_objects/examples/export_objects_response.yaml:3]: Sure. will trim it.

File: docs/openapi/saved_objects/saved_objects.yml (hunk 3 above)
Comment (zhyuanqi) [reply docs/openapi/saved_objects/saved_objects.yml:530]: Sure let me make change of the title

**Review by BionIT — DISMISSED** [review]

**Review by BionIT — APPROVED** [review]

**Review by ZilongX — APPROVED** [review]

**Review by zhongnansu — APPROVED** [review]

**Pull request merged**